include/rocksdb/secondary_cache.h

```diff
@@ -68,11 +68,14 @@ class SecondaryCache : public Customizable {
   // Lookup the data for the given key in this cache. The create_cb
   // will be used to create the object. The handle returned may not be
   // ready yet, unless wait=true, in which case Lookup() will block until
-  // the handle is ready. is_in_sec_cache is to indicate whether the
-  // handle is possibly erased from the secondary cache after the Lookup.
+  // the handle is ready.
+  // The erase_handle param should be ignored by the implementation
+  // on non-volatile media
+  // is_in_sec_cache is to indicate whether the handle is possibly erased
+  // from the secondary cache after the Lookup.
   virtual std::unique_ptr<SecondaryCacheResultHandle> Lookup(
       const Slice& key, const Cache::CreateCallback& create_cb, bool wait,
-      bool& is_in_sec_cache) = 0;
+      bool erase_handle, bool& is_in_sec_cache) = 0;
```

Comment:
Perhaps change the argument name to something like `advice_erase` or something like that.